features/LIBRARY.feature

```diff
@@ -0,0 +1,93 @@
+Feature: LIBRARY
+  Scenario: Editing books in the table
+    Given We are logged in as admin and we are in the LIBRARY section
+    When We click the edit button
+    And We change the title and / or author
+    And Confirm with the Ok button
+    Then The author and / or title for this book are changed
```

Comment:
To samo co wyżej musisz wybrać co konkretnie zmieniłeś albo napisać nowy test zmieniający tylko jedną rzecz